src/components/Form.js

```diff
@@ -14,17 +14,18 @@ export default function Form() {
     const id = `item${Math.random()}`;
     dispatch(addBook({ id, title, author }));
 
-    await dispatch(getBooks({ id, title, author }));
+    await dispatch(fetchBooks({ id, title, author }));
     setTitle('');
     setAuthor('');
   };
 
   return (
     <div className="Form">
+      <div className="vert-line2" />
       <span className="add-newbook"> ADD NEW BOOK</span>
       <form action="" onSubmit={addSubmit}>
         <input className="input-title" placeholder="Book title" type="text" value={title} onChange={(e) => setTitle(e.target.value)} />
-        <input className="input-author" placeholder="Book author" type="text" value={author} onChange={(e) => setAuthor(e.target.value)} />
+        <input className="input-author" placeholder="author" type="text" value={author} onChange={(e) => setAuthor(e.target.value)} />
         <button className="submit-btn" type="submit">
           ADD BOOK
         </button>
```

Comment:
- [ ] Good job working on this, but I wasn't able to see the complete styling of the page because the API call to add and fetch books throws an error. Please kindly fix this error so all the styling can be reviewed.

### Add Books error
```json
{
    "error": {
        "status": 400,
        "message": "'item_id' is required. 'category' is required."
    }
}
```

### Fetch books error
```json
{"error":{"status":500,"message":"Cannot read property 'books' of undefined"}}
```